code/modules/mob/living/carbon/human/human.dm

```diff
@@ -1482,7 +1482,7 @@
 	Jitter(jitter_duration)
 
 /mob/living/carbon/human/proc/asthma_attack()
-	if(disabilities & ASTHMA && !(M_NO_BREATH in mutations) && !(has_reagent_in_blood(ALBUTEROL)))
+	if(disabilities & ASTHMA && !(M_NO_BREATH in mutations) && !(species && species.flags & NO_BREATHE) && !(has_reagent_in_blood(ALBUTEROL)))
```

Comment:
```suggestion
	if(disabilities & ASTHMA && !(M_NO_BREATH in mutations) && !(species?.flags & NO_BREATHE) && !(has_reagent_in_blood(ALBUTEROL)))
```

Reply:
The only way that species can be null is when the human is being deleted, and at that point it shouldn't be life()ing